Bugzilla/DB/Mysql.pm

```diff
@@ -332,10 +332,17 @@ sub bz_setup_database {
   if ($self->utf8_charset eq 'utf8mb4') {
     my %global = map {@$_}
       @{$self->selectall_arrayref(q(SHOW GLOBAL VARIABLES LIKE 'innodb_%'))};
+
+    # Newer versions of MySQL the default value for innodb_file_format is Barracuda
```

Comment:
```suggestion
    # In newer versions of MySQL the default value for innodb_file_format is Barracuda
```

Also, if you could specify a rough range (is "in MySQL 8+" correct?) that would help for readers of this comment a few years from now, when "newer version of MySQL" means something different :)